packages/admin/admin/src/FinalForm.tsx

```diff
@@ -131,6 +129,10 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
         const submit = React.useCallback(
             (event: any) => {
                 event.preventDefault(); //  Prevents from reloading the page with GET-params on submit
+                if (saveBoundaryApi) {
+                    // if we are inside a SaveBoundary, save the whole SaveBoundary
+                    return saveBoundaryApi.save();
+                }
```

Comment:
So this doesn't cause an infinite submission loop?

Reply:
no, because Savable doesn't trigger the `<form` submit, it uses final-form api.